### DISCUSSION TITLE
Your uploaded proof has already been used previously

### DISCUSSION THREAD

**Discussion by shanirinacohen** [community/community#58442] [GitHub Education]
### Select Topic Area

Bug

### Body

Hi i got this response for GitHub:" Your uploaded proof has already been used previously"

My husband (dviroscohen) has a Github account, so i thought to use his account, so I added my academic mail to his account and used my Student Id, but got a message that it's not him. So I removed my mail from there and then i've created a new account with my mail and my academic mail, but now when i'm trying to send the student id it says that it was already used for another account.
But my husband removed it from his account! So it's like no one is using it. I need fast support here.

I commented here but got no response for a week:
https://github.com/orgs/community/discussions/52990

Thanks.

**Comment by the-officialjosh** [comment]
I understand your frustration, but I'm sorry to inform you that it is highly unlikely that this issue can be resolved. The student pack provided by GitHub has strict guidelines, and even students who meet all the requirements sometimes face challenges in getting accepted for various reasons. In your specific case, the platform is designed to prevent the use of a student email on multiple accounts to access these benefits. While I understand that you may have made an error by adding the email to your husband's account, it is unlikely that it can be resolved to allow you to use the same email again for your student account. I apologize for any inconvenience this may cause.

**Reply by shanirinacohen** [reply]
Thanks for the response. Are you an official GitHub rep? There is always a backdoor. I'm not using the same account twice. I removed any use of it from the first account, what was originally an innocent mistake, and added it to the correct one.

**Reply by the-officialjosh** [reply]
You could keep on trying however my reply was based on the countless number of issues here which you could search if you want of similar issues. Anyways I am not saying you should lose hope on the matter it is still worth trying though

**Reply by shanirinacohen** [reply]
Thank you.

**Reply by the-officialjosh** [reply]
Mark that top answer as the reply please for others with similar issues to be directed here

**Reply by shanirinacohen** [reply]
I still want to get an official response from GitHub..

**Reply by the-officialjosh** [reply]
Oh ok fine, no problem 👍🏿

**Comment by samus-aran** [comment]
Hi @shanirinacohen 👋 

This particular rejection reason states that you must supply a different image of your submitted evidence (or another piece of evidence altogether). 

Let us know here if you need further assistance 👍

**Reply by Lauralx** [reply]
Es correcto intente con otro correo y no me siguio apareciendo el mismo error, Gracias!

**Comment by tdaudi** [comment]
Hello @shanirinacohen !
To bypass this github behavior you just have to put you image in an online jpg size reducer. As the size of the file is different, github understand it as a different image. 
I was facing the same issue, but this solution worked for me.

**Reply by MRdima98** [reply]
Thanks man it works!
